Neos.Neos/NodeTypes/Document/Shortcut.yaml

```diff
@@ -46,3 +46,15 @@
             setTargetModeIfNotEmpty:
               property: 'targetMode'
               handler: 'Neos.Neos/Inspector/Handlers/ShortcutHandler'
+
+    httpStatusCode:
+      type: int
+      defaultValue: 303
+      ui:
+        label: i18n
+        inspector:
+          # change this to edit the status code in the inspector
+          hidden: true
+          group: 'document'
+          editor: 'Neos.Neos/Inspector/Editors/TextFieldEditor'
```

Comment:
I would offer here a selectbox with the different values

Reply:
According to the Poll in the issue we will be using a general setting so I removed the whole thing